Desktop/components/JASP/Widgets/MainPage.qml

```diff
@@ -255,14 +255,25 @@ Item
 				}
 			}
 
-			property var urlWhitelist : ['www.youtube.com', 'www.youtu.be', 'player.vimeo.com', 'www.bilibili.com', 'v.qq.com'];
+			property var urlWhitelist : ['www.youtube.com', 'www.youtu.be', '*.vimeo.com', '*.vimeocdn.com', '*.akamaized.net', '*.bilibili.com', '*.hdslb.com', '*.qq.com', '*.smtcdns.com'];
```

Comment:
is `smtcdns.com` a chinese content delivery network or something?
The `akamaized` one I can find some info on, just like for `hdslb.com`

Reply:
This is some video cdn server which vimeo or other sever used.

Reply:
<img width="1342" alt="image" src="https://github.com/jasp-stats/jasp-desktop/assets/29062713/6b4e1ae5-f3de-42c9-a45b-669718553a7a">
Ah yeah for `qq`